flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -242,6 +243,14 @@ public String getOperatorName() {
         return operatorName;
     }
 
+    public String getOperatorDesc() {
+        return null == operatorDesc ? operatorName : operatorDesc;
```

Comment:
I'd move this logic to call-site. Either while creating the StreamNode or while reading the StreamNode.